routing.md

```diff
@@ -277,9 +277,35 @@ Why is such indirection useful?
 * It lets the mediator remain uncommitted to whether the next receiver is another mediator or not. This may provide flexibility in some routing scenarios.
 * It lets the mediator change the size of the message by adding or subtracting noise from the content. 
 * It allows for dynamic routing late in the delivery chain.
- 
+
 These last two characteristics are the foundation of mix networking feature for DIDComm. That feature is the subject of a different RFC; here we only note the existence of the optional feature. 
 
 
-### DID Docs and Routing
+### DID Document Service Endpoint
+DIDComm DID Document endpoints have the following format:
+
+```json
+{
+    "id": "did:example:123456789abcdefghi#didcomm-1",
+    "type": "DIDComm",
+    "publicKey": "did:example:123456789abcdefghi#key-1",
```

Comment:
Although I understand this has been the DIDComm convention from Aries, I am curious about specifying publicKey in this way.

- I've seen others using "verification method" sections in the DID document to specify key usage. e.g., `keyAgreement` seems similar to this usage.
- Is this defined the same way as public keys defined in "verification method" sections? Example: https://www.w3.org/TR/did-core/#authentication

Reply:
@dhh1128 is this consistent with your argument about not designating keys in the service block?

Reply:
There is an example in the DID core spec for Identity Hub that also has the publicKey reference.

It seems the higher-level question is: am I defining the DIDComm publicKey for a particular agent or am I defining "my" DIDComm publicKeys in general.

There seem to be examples of both in the DID core spec.
For the latter, see above comment. For the former, see the identity hub example in https://www.w3.org/TR/did-core/#example-16-various-service-endpoints

So both usages likely have their place.

Reply:
See my comment [below](https://github.com/decentralized-identity/didcomm-messaging/pull/48#issuecomment-626964010).

Reply:
resolving, as this has been removed.